pulsar-broker/src/test/java/org/apache/pulsar/broker/service/SystemTopicBasedTopicPoliciesServiceTest.java

```diff
@@ -457,4 +459,27 @@ public void testGetTopicPoliciesWithCleanCache() throws Exception {
 
         result.join();
     }
+
+    @Test
+    public void testConfig() throws Exception {
```

Comment:
The test can also get passed without this change.

Reply:
Updated